<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="mnuOpen"/>
    <addaction name="mnuOpenDatabase"/>
    <addaction name="mnuClose"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mnuOpen"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="dockVerticalLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="mnuOpen">
   <property name="icon">
    <iconset resource="resurses.qrc">
     <normaloff>:/img/addGroup_24.png</normaloff>:/img/addGroup_24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Отркыть</string>
   </property>
  </action>
  <action name="mnuClose">
   <property name="text">
    <string>&amp;Закрыть</string>
   </property>
  </action>
  <action name="mnuOpenDatabase">
   <property name="text">
    <string>Открыть базу данных</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resurses.qrc"/>
  <include location="resurses.qrc"/>
 </resources>
 <connections/>
</ui>
